packages/dashboard/test/helpers.ts

```diff
@@ -165,7 +165,8 @@ export function expectLayout(dashboard: HTMLElement, layout: Array<Array<number
       if (!element) {
         actualRow.push(null);
       } else {
-        actualRow.push(parseInt(element.id.replace('item-', '')));
+        // TODO: Just use a number for all test item IDs
```

Comment:
Should we add this to an improvements list or create an issue for this?

Reply:
It's a small chore, shouldn't need an issue

Reply:
Fixed here https://github.com/vaadin/web-components/commit/274c8d93c8d32878c1dd12f7bbec4bdb22f6b109